src/actions/adjust/ImproveAction.ts

```diff
@@ -19,7 +20,7 @@ class ImproveAction extends Action {
    * @description The improve mode.
    * @param {Qualifiers.ImproveMode | string} value
```

Comment:
do we want to add what value can be for reference docs?

Reply:
Already documented here 
https://staging.cloudinary.com/documentation/sdks/js/cloudinary-js-base/Qualifiers.ImproveMode.html

and we're using strict qualifiers in our docs.